packages/resource-deployment/templates/rest-api-templates/model-accessibility-insight-service-scan-api-api.template.json

```diff
@@ -241,7 +281,28 @@
             "apiVersion": "2019-01-01",
             "dependsOn": [
                 "[resourceId('Microsoft.ApiManagement/service/apis', parameters('apimServiceName'), 'accessibility-insight-service-scan-api')]",
-                "[resourceId('Microsoft.ApiManagement/service/apis/schemas', parameters('apimServiceName'), 'accessibility-insight-service-scan-api', '5d65a7add888c31128385485')]"
+                "[resourceId('Microsoft.ApiManagement/service/apis/schemas', parameters('apimServiceName'), 'accessibility-insight-service-scan-api', '5d704a4ef518fc0430710495')]"
+            ]
+        },
+        {
+            "properties": {
+                "value": "[concat('<policies>\r\n\t<inbound>\r\n\t\t<base />\r\n\t\t<set-backend-service id=\"apim-generated-policy\" backend-id=\"', parameters('functionName'), '\" />\r\n\t</inbound>\r\n\t<backend>\r\n\t\t<base />\r\n\t</backend>\r\n\t<outbound>\r\n\t\t<base />\r\n\t</outbound>\r\n\t<on-error>\r\n\t\t<base />\r\n\t</on-error>\r\n</policies>')]",
+                "format": "xml"
+            },
+            "name": "[concat(parameters('apimServiceName'), '/accessibility-insight-service-scan-api/getScanBatch/policy')]",
+            "type": "Microsoft.ApiManagement/service/apis/operations/policies",
+            "apiVersion": "2019-01-01",
+            "dependsOn": [
+                "[resourceId('Microsoft.ApiManagement/service/apis/operations', parameters('apimServiceName'), 'accessibility-insight-service-scan-api','getScanBatch')]"
+            ]
+        },
+        {
+            "properties": {},
+            "name": "[concat(parameters('apimServiceName'), '/starter/accessibility-insight-service-scan-api')]",
```

Comment:
have the api registered in unlimited product

Reply:
It is already a part of both starter and unlimited. I will remove the addition of this product to starter.